injected/src/features/broker-protection/actions/navigate.js

```diff
@@ -17,15 +17,18 @@ export function navigate(action, userData) {
         return urlResult;
     }
 
-    const codeToInjectResponse = getSupportingCodeToInject(action);
-    if (codeToInjectResponse instanceof ErrorResponse) {
-        return codeToInjectResponse;
-    }
-
     const response = {
         ...urlResult.success.response,
-        ...codeToInjectResponse.success.response,
     };
 
+    if (action.injectCaptchaHandler) {
```

Comment:
We were calling this regardless of the presence of injectCaptchaHandler. Seemed better to do it conditionally.